src/ServerConfigParams.ts

```diff
@@ -111,3 +144,17 @@ export function getServerConfig(): ServerConfigParams {
 
   return configuration;
 }
+
+export {
```

Comment:
Is there any specific reason for changing the way we export the functions/constants?

Reply:
No, I'm just using the way I have seen in some recent PR's I've reviewed.